app/index.html

```diff
@@ -7,6 +7,7 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
   <title>Telegram Web</title>
+  <meta property="description" content="Welcome to the Web application of Telegram messenger. See https://github.com/zhukov/webogram for more info.">
```

Comment:
Maybe:
"Official web version of Telegram - most heavily encrypted and can self-destruct messenger"

Reply:
done